src/main/java/digital/slovensko/autogram/core/eforms/UpvsEFormResources.java

```diff
@@ -73,4 +74,16 @@ public boolean findResources() throws XMLValidationException {
 
         return true;
     }
+
+    private byte[] getRemoteResource(String url) {
+        byte[] bytes = getResource(url);
+        if (bytes == null)
+            return null;
+
+        String s = new String(bytes, ENCODING);
+        if (s.contains("<html>") && s.contains("<title>Nedostupnosť portálu</title>"))
```

Comment:
Neotocime tu podmienku na to čo tam očakávame a nie na to čo tam nemá byť? Ak si Slovensko.sk vymyslí novú error hlášku tak sa to rozbije.

Reply:
Tak teoreticky stačí, aby sme detekovali, či sa tam nachádza `<html>` keďže to by sa vo formulároch nemalo nachádzať vôbec. (pri vytvorení novej hlášky by mohli aj upraviť status kód 😂)

Reply:
`<html>` sa práveže nachádza v html xslt transformáciach, takže iba to stačiť nebude. tú hlášku asi nebudú meniť často, ale môžeme dať keď tak OR na výskyt `<!DOCTYPE html>` a `<html>` s tým nadpisom.

Reply:
Nie je to zbytočné? `<!DOCTYPE html>` bohato stačí.

Reply:
no obrátiť to nevieme imho, lebo sťahovať môžeme kadečo a nevieme presne povedať, čo by to malo obsahovať. `<!DOCTYPE html>` stačí v exampli, ktorý máme k dispozícii, ale tiež je tam možnosť, že budú niečo meniť a zrazu sa im bude servovať html bez tejto deklarácie.

Reply:
no obrátiť to nevieme imho, lebo sťahovať môžeme kadečo a nevieme presne povedať, čo by to malo obsahovať. `<!DOCTYPE html>` stačí v exampli, ktorý máme k dispozícii, ale tiež je tam možnosť, že budú niečo meniť a zrazu sa im bude servovať html bez tejto deklarácie.

Reply:
To máš pravdu, pri tomto si nemôžme byť 100% istý, takže to čo si navrhol je asi najlepšie riešenie.